build.gradle

```diff
@@ -16,24 +16,24 @@
 import org.gradle.api.internal.plugins.osgi.OsgiHelper
 
 buildscript {
-  ext.kotlinVersion = '1.2.51'
+  ext.kotlinVersion = '1.3.31'
   repositories {
 	maven { url "https://repo.spring.io/plugins-release" }
   }
   dependencies {
 	classpath 'org.springframework.build.gradle:propdeps-plugin:0.0.7'
 	classpath 'io.spring.gradle:spring-io-plugin:0.0.4.RELEASE'
-	classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.0'
+	classpath "com.github.jengelman.gradle.plugins:shadow:4.0.2"
+	classpath "me.champeau.gradle:jmh-gradle-plugin:0.4.8"
```

Comment:
could the jmh plugin dependency and application be moved into `benchmarks` module build.gradle?

Reply:
I think i tried but gradle wants to see buildscript dependencies in the top level project or something like that.